### DISCUSSION TITLE
[Feature] Hide suggested contacts when share album

### DISCUSSION THREAD

**Discussion by Christopher-Songs** [immich-app/immich#12746] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Is there a way to hide suggested contacts when user share an album? Because my immich instance was open to several friends, and they didn't know each other. If there was a switch to turn it off, I'd really appreciate it.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4408
